nvflare/lighter/impl/signature.py

```diff
@@ -27,14 +27,20 @@ class SignatureBuilder(Builder):
     """
 
     def build(self, project: Project, ctx: dict):
+        root_pri_key = ctx.get("root_pri_key")
+
+        overseer = project.get_participants_by_type("overseer", first_only=True)
+        dest_dir = self.get_kit_dir(overseer, ctx)
+        signatures = sign_all(dest_dir, root_pri_key)
+        json.dump(signatures, open(os.path.join(dest_dir, "signature.json"), "wt"))
```

Comment:
just a minor improvement, these three lines are repeating for overseer, server and client
maybe we can consider make a method?

Reply:
Good catch.  Thanks.  Just pushed another commit with refactored function.